docs/stack/insight/tutorials/insight-stream-consumer.md

```diff
@@ -0,0 +1,199 @@
+---
+title: "Manage streams and consumer groups in RedisInsight"
+linkTitle: Streams
+description: Learn how to manage streams and consumer groups in RedisInsight
+weight: 1
+---
+
+A _stream_ is an append-only log file. 
+When you add data to it, you cannot change it. 
+That may seem like a disadvantage; however, a stream serves as a log or single source of truth. 
+It is used as a buffer between processes that work at different speeds and do not need to know about each other. 
```

Comment:
```suggestion
It can also be used as a buffer between processes that work at different speeds and do not need to know about each other. 
```

Suggesting rewording as this is a second, distinct, use case.